### PR TITLE
Add support for Cloud Run service

### PR DESCRIPTION
**Pull request by yegle**
Fixes https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/117

### CODE REVIEW

**Comment by yegle** [comment]
Please hold merging this PR. I think I should get https://github.com/open-telemetry/opentelemetry-go-contrib/pull/455 merged first.

**Comment by yegle** [comment]
I've updated the PR based on the detector added in open-telemetry/opentelemetry-go-contrib#455.

This PR is now ready to review. PTAL.

File: exporter/metric/constants.go [hunk 1]
```diff
@@ -52,11 +52,17 @@ const (
 	K8SKeyPodName        = "k8s.pod.name"
 	K8SKeyDeploymentName = "k8s.deployment.name"
 
+	// Cloud Run
+	ServiceNamespace  = "service.namespace"
```
Comment (dashpole) [inline exporter/metric/constants.go:56]: To be consistent with other keys, use `ServiceKeyNamespace` and similar below.

File: exporter/metric/constants.go [hunk 2]
```diff
@@ -52,11 +52,17 @@ const (
 	K8SKeyPodName        = "k8s.pod.name"
 	K8SKeyDeploymentName = "k8s.deployment.name"
 
+	// Cloud Run
```
Comment (dashpole) [inline exporter/metric/constants.go:55]: These constants don't appear to be cloud-run specific.

**Comment by yegle** [comment]
Updated per review comments. PTAL.

**Review by dashpole — APPROVED** [review]
lgtm

**Pull request merged**